doc/example_code/how_to_examples/joystick_controller.rst

```diff
@@ -0,0 +1,169 @@
+.. _example-code:
+
+How To Get Connected Controllers
+================================
+We can get a list of connected controllers by using :code:`arcade.get_joysticks()`.
+
+.. code-block:: python
+
+      joysticks = arcade.get_joysticks()
+      if joysticks:
+          self.joystick = joysticks[0]
+          self.joystick.open()
+      else:
+          print("There are no joysticks.")
+          self.joystick = None
+
+Joystick Values
```

Comment:
This section should be using the `leftx`, `lefty`, `rightx`, and `righty` values for joystick positions. The plain x/y values are for the face buttons.

Reply:
Would it be something like this? 
`self.object.change_leftx = self.controller.leftx`
 `self.object.change_lefty = self.controller.lefty`